### PR TITLE
README: Add ansible-lint setup

### PR DESCRIPTION
**Pull request by mnecas**
Add to readme how to setup ansible-lint for engine

### CODE REVIEW

File: README.adoc [hunk 1]
```diff
@@ -608,3 +608,18 @@ Build minimal rpm package for Firefox::
...
+
+=== Ansible Lint
+
+To use ansible-lint locally you need to install it from PyPI in a python virtualenv/venv, e.g.:
+```
+sudo dnf install python3-virtualenv
+mkdir -p $HOME/venv
+python3 -m $HOME/venv ansible-lint
+. $HOME/venv/ansible-lint/bin/activate
+pip3 install ansible-lint
```
[the remark this answers is not in the document]
Comment (mnecas) [reply README.adoc:620]: We could add `ansible-lint<=7.0.0` but currently we don't have this requirement in any of our CI

[the remark this answers is not in the document]
Comment (mwperina) [reply README.adoc:620]: So can we add something like `ansible-list >= 6.0.0, < 7.0.0

[the remark this answers is not in the document]
Comment (mnecas) [reply README.adoc]: done

[the remark this answers is not in the document]
Comment (mnecas) [reply README.adoc]: I have also set the `"ansible-lint>=6.0.0,<7.0.0"` to the collection CI and updated the engine patch to have this version

[the remark this answers is not in the document]
Comment (mwperina) [reply README.adoc]: +1

**Review by mwperina — APPROVED** [review]
+1

**Pull request merged**